### ISSUE TITLE
Possibly not updating LLVM submodule

### ISSUE THREAD

**Issue by youngar** [llvm/circt-www#5]
The website is currently [unable to successfully](https://github.com/llvm/circt-www/runs/1923711256?check_suite_focus=true#step:9:142) run `make circt-doc`.  This target is building fine in the normal repository.

```
/home/runner/work/circt-www/circt-www/circt_src/include/circt/Dialect/SV/SV.td:38:22: error: Value 'cppNamespace' unknown!
  let cppNamespace = "::circt::sv";
```

It seems like this might be due to using an outdated `mlir-tablegen` by the website CI.

The stage [Get LLVM Hash](https://github.com/llvm/circt-www/runs/1923711256?check_suite_focus=true#step:6:1) seems suspect:
```
Run echo "::set-output name=hash::$(git rev-parse @:./llvm)"
  echo "::set-output name=hash::$(git rev-parse @:./llvm)"
  shell: /bin/bash --noprofile --norc -e -o pipefail {0}
fatal: path 'llvm' does not exist in '@'
```

**Comment by xgupta** [comment]
Thanks, @youngar If you understand the issue please update main.yml accordingly. I make this website run by hit & trail :-)

**Comment by youngar** [comment]
Due to a mistake in the caching logic for LLVM we were not rebuilding it when necessary.  This has been fixed, and the website is working again.  Fixed across a couple commits starting at 998c8f6fc5d3199dae0fd8c29dd6fc0fdaaf2f16.

**Comment by xgupta** [comment]
Thanks!
